history/characters/10000_orc.txt

```diff
@@ -335,7 +335,7 @@
 		culture=scourge religion=death_god
 		trait = being_undead
 		remove_trait = creature_orc
-		trait = creature_lich
+		effect = { add_trait_force_tooltip = creature_lich }
```

Comment:
I'm sorry to bother you with such a thing since everything works.
But I found out it may work like this:
```suggestion
		add_trait = creature_lich
```
This is much clearer.
I would do it myself, but I'm not allowed to push on this branch.

Reply:
No problem, I also agree with that approach! Worked perfectly
![image](https://user-images.githubusercontent.com/35965798/149621891-23d2757c-b191-414f-a4f2-526c3806a42c.png)